### PR TITLE
Corrections and clarifications

### PR DESCRIPTION
**Pull request by Eystein-Thanisch**
From beginning to look through the new spreadsheet of the database, I have identified a few new owl:sameAs links that could be made and some further minor spelling corrections. A few of these involve a bit of interpretation, so you might want to check them yourself.

### CODE REVIEW

File: LL/ciarraige_luachra.ttl [hunk 1]
```diff
@@ -19,107 +19,125 @@
...
+    rel:childOf <#MaeleTuile>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Rectabrat>.
 
 <#MaeleTuile>
     a foaf:Person;
     foaf:name "Maele Tuile";
     foaf:name "Mael Tul";
-    rel:childOf <#Aidloga>.
+    rel:childOf <#Aidloga>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#MaeliTuile>.
```
Comment (cyocum) [inline LL/ciarraige_luachra.ttl]: These are pointing to a completely different name.  Are you sure this is correct?

File: LL/cland_shechnasaig.ttl [hunk 2]
```diff
@@ -175,38 +175,38 @@
...
     a foaf:Person;
-    foaf:Name "Dunechaid";
+    foaf:name "Dunechaid";
     rel:childOf <#Dunechaid>.
 
 <#Condalaig-47b1f030>
     a foaf:Person;
     irishRel:numChild 3;
-    foaf:Name "Condalaig";
+    foaf:name "Condalaig";
```
Comment (cyocum) [inline LL/cland_shechnasaig.ttl]: nom. is Condalach. Could you quickly add it :)

**Comment by cyocum** [comment]
Other than the one thing that I pointed out. Looks good!

**Comment by Eystein-Thanisch** [comment]
Ok, Condalach added. But I fail to see how the owl:sameAs <
http://example.com/LL/ciarraige.ttl#MaeliTuile> link points to someone with
a different name. They have the same name and they appear in a very similar
pedigree?

**Comment by cyocum** [comment]
I had deleted that comment because I was wrong.  Sorry, you must have gotten an email when I added it in error.

**Pull request merged**